.github/workflows/docs/frontend/frontend_runtime_application_manual_e2e_run.md

```diff
@@ -0,0 +1,84 @@
+# Default workflow for running E2E tests when a PR comment is left
+
+
+This is the [default workflow](../../frontend_runtime_application_manual_e2e_run.yml) that is leveraged to trigger E2E tests for for an `application runtime`. Specifically, when a PR is opened and a comment of `/run-e2e-tests` is left, it will trigger an E2E test run.
+
+## Inputs
+
+This action takes the following inputs:
+
+| Name                        | Type    | Default                      | Required  | Description                                                                            |
+| --------------------------- | ------- | ---------------------------- | --------- | -------------------------------------------------------------------------------------- |
+| `fallback_runner`            | String  | False                        | False      | If true will leverage ubuntu-latest, otherwise will fall back to the J1 in-house runner
+| `use_magic_url`              | Boolean | True                         | False      | Deploy to dev via a query param, required for normal SPAs
+| `use_e2e`                    | Boolean | False                        | False      | Run E2E test, in most case we want this
+| `e2e_filter_tags`            | String  |                              | False      | Tests will be filtered based on the tags defined here
+| `e2e_containers`             | String  | '["1"]'                      | False      | The number of tests that you want Cypress to run in parallel. For example to run your tests via 3 containers you would set the value to the following: '["1", "2", "3"]'.
+| `e2e_pass_on_error`          | Boolean | False                        | False      | Pass the workflow even if the E2E test fail
+| `e2e_artemis_config_path`    | String  | cypress/artemis-config.yaml  | False      | Used to determine the path to the artemis config file
+| `spec_to_run`                | String  | cypress/e2e/**/*.feature     | False      | Used to determine which test to run
+| `magic_url_route`            | String  | '/'                          | False      | The relative route the magic url should go to
+| `auto_cancel_after_failures` | Number  | 2                            | False      | Cancel the tests after this many failures
+                                                                           
+## Secrets
+
+This action takes the following secrets:
+
+| Name                        | Required  | Description                               |
+| --------------------------- | --------- | ----------------------------------------- |
+| `NPM_TOKEN`                 | True      | A J1 npm.com Publish token
+| `CYPRESS_RECORD_KEY`        | False     | The record key associated with the project in Cypress.
```

Comment:
Why are these not required?

Reply:
Good call. They weren't initially required because the E2E flow was a part of the overall PR flow, and not ever repo runs E2E tests. Now that these are split out they should be required. These have since [been updated](https://github.com/JupiterOne/.github/pull/112/commits/b9f0929498cfeda99b7992cd086b993b0f830b4b).